src/Sulu/Component/Webspace/PortalInformation.php

```diff
@@ -375,11 +375,17 @@ public function setUrlExpression($urlExpression)
     /**
      * Calculate the length of the host part of the URL.
      *
+     * @param bool $withPort
+     *
      * @return int
      */
-    private function getHostLength()
+    private function getHostLength($withPort = false)
```

Comment:
@danrot  the getHost use the default without the port. But here I'm not sure where getHost is used and if it should always return without the domain. symfony itself seems to have no port specific routing